doc/install.rst

```diff
@@ -12,10 +12,11 @@ Installation
 #. Add the callback processor to your URL router::
 
       # urls.py
-      from django.conf.urls import include, url
+      from django.conf.urls import include, path
 
       urlpatterns = [
-          url('^payments/', include('payments.urls'))]
+          path('payments/', include('payments.urls')),
```

Comment:
I think you may need `re_path` here as `path` does an exact match (instead of matching the prefix).

Reply:
This matches exactly the same as the previous code. `re_path` is only needed if we use a regex instead of a plain string (that's not the case here).

Reply:
Sorry, you're right, missed the `include` as the second parameter.